src/puppet-js-view-model-tree.html

```diff
@@ -91,6 +91,10 @@
       var puppet = listener.getPuppetClient();
       if (puppet) {
         clone.querySelector('juicy-jsoneditor').json = puppet.obj;
+        // passover changes to Puppet
+        clone.querySelector('juicy-jsoneditor').addEventListener('change', function(e) {
+          puppet.puppet.handleLocalChange(e.detail.patches);
```

Comment:
I guess it's not a good idea to work on this anymore at the moment. Because:

1. Merge as is to support Puppet. Because Puppet will never have proxies and using its internals isn't problematic, since it will never be updated.

2. East or west, this will need a major change once `palindrom-connection` is adopted. So why waste time now to cover for upcoming proxies and then spend time again to handle the movement from `puppet-client` to `palindrom-connection`. It will take 200% of the time.

I don't mind working on this at all, but I think current state works perfectly but enhancements to it would be discarded soon.